benchmark/sparse_blas/operations.cpp

```diff
@@ -566,6 +569,98 @@ class LookupOperation : public BenchmarkOperation {
 };
 
 
+bool validate_factorization(const Mtx* input, const Mtx* factors)
+{
+    const auto host_exec = input->get_executor()->get_master();
+    const auto host_input = gko::clone(host_exec, input);
+    const auto host_factors = gko::clone(host_exec, factors);
```

Comment:
`make_temporary_clone`? But I guess the performance is not important here.

Reply:
good point, originally I wanted to sort the input, but the ordering is always guaranteed by the algorithms